### ISSUE TITLE
Update or remove obsolete articles

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#663]
- [x] https://www.dartlang.org/articles/language
    - [ ] https://www.dartlang.org/articles/language/await-async - #512 suggests combining async/generator docs.
    - [ ] https://www.dartlang.org/articles/language/beyond-async - #512 suggests combining async/generator docs.
    - [x] https://www.dartlang.org/articles/language/emulating-functions - #600 has a list of changes to make.
    - [x] https://www.dartlang.org/articles/language/mixins [UPDATED: #978]
- [ ] https://www.dartlang.org/articles/libraries
    - [ ] https://www.dartlang.org/articles/libraries/broadcast-streams
    - [ ] https://www.dartlang.org/articles/libraries/converters-and-codecs
    - [x] https://www.dartlang.org/articles/libraries/creating-streams - PR: #1180
    - [x] https://www.dartlang.org/articles/libraries/serialization - PRs: #253, #1034  
    - [ ] https://www.dartlang.org/articles/libraries/zones - issues: #2151, #2150, #2149, #2148, #2147
- [ ] https://www.dartlang.org/articles/server - The path used to be /articles/dart-vm. To see the pre-move history, look at a [pre-move source tree](https://github.com/dart-lang/site-www/tree/b5a930423afd5536118f10c72db213333955c6a1/src/_articles/dart-vm).)
    - [x] https://www.dartlang.org/articles/server/benchmarking - PR: #1034 
    - [x] https://www.dartlang.org/articles/server/io - #275 - PR: #1110 
    - [x] https://www.dartlang.org/articles/server/native-extensions - was blocked on https://dart-review.googlesource.com/c/sdk/+/63260; in May, Siva said no updates seemed to be needed.
    - [x] https://www.dartlang.org/articles/dart-vm/numeric-computation - PR: #1034 
    - [x] https://www.dartlang.org/articles/dart-vm/reflection-with-mirrors - #32; don't plan to update, since mirrors aren't recommended in most cases.
    - [x] https://www.dartlang.org/articles/dart-vm/simd - PR: #1034

**Comment by kwalrath** [comment]
Per the discussion on dart-lang/sdk#33960, here's my proposal:

I get each article reviewed by the most knowledgable people I can find on the subject, and then add a note at the top of each article that addresses the following points:

* Who should read (or not read) the article, and why (or why not).
* What's changed since the article was written.
* What's still true.
* What's important.
* Where else they can find information on this topic.

For articles that have important content, we should consider either updating the article or writing more maintainable documentation somewhere — perhaps in the dart-lang/sdk repo, where engineers working on the source code are more likely to encounter and update it.

**Comment by atsansone** [comment]
This list of links applied to prior infrastructure. The degree of changes make this issue obsolete. If this is still a going concern, please create a new issue and update relevant information. Thanks!
